ui/src/features/search/searchSchemas.ts

```diff
@@ -148,6 +148,65 @@ export const serviceSchema = () => {
 		);
 };
 
+export const supportedCicdTools = [
+	{
+		id: "aws_codebuild",
+		displayName: "AWS CodeBuild",
```

Comment:
These are the names of services. I imagine that is consistent across languages. Should these be translatable?

Reply:
These are all proper nouns, so I don't _think_ they would be translated.  Let's leave them untranslated -- we can add translations later if we find a counterexample.